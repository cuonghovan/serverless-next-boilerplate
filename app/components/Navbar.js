import React from "react";
import Link from "next/link";

export default () => (
  <nav className="nav">
    <ul>
      <li>
        <Link href="/">
          <span>home</span>
        </Link>
      </li>
      <li>
        <Link href="/a">
          <span>Page a</span>
        </Link>
      </li>
      <li>
        <Link href="/b">
          <span>Page b</span>
        </Link>
      </li>
    </ul>
  </nav>
);
