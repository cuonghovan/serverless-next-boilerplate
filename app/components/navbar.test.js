/*global shallow mount*/
import React from "react";
import Link from "next/link";
import Navbar from "./Navbar";

describe("search-bar", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should contains three link", () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper.find(Link)).toHaveLength(3);
  });
});
