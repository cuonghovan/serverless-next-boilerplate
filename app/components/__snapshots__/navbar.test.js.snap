// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search-bar renders correctly 1`] = `
<nav
  className="nav"
>
  <ul>
    <li>
      <Link
        href="/"
      >
        <span>
          home
        </span>
      </Link>
    </li>
    <li>
      <Link
        href="/a"
      >
        <span>
          Page a
        </span>
      </Link>
    </li>
    <li>
      <Link
        href="/b"
      >
        <span>
          Page b
        </span>
      </Link>
    </li>
  </ul>
</nav>
`;
